01-HTML-CSS-Basics/2. CSS/2.2 Selectors and properties/index.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <header>
+        <p>Este es el HEAD</p>
+        <br/>
```

Comment:
br tags should only be use for linebreaks. If you need some space for a styling purpose, try to use margin, paddings css properties.